salt/salt.spec

```diff
@@ -351,9 +351,13 @@ BuildRequires:  python-futures >= 2.0
 BuildRequires:  python-msgpack-python > 0.3
 BuildRequires:  python-psutil
 BuildRequires:  python-requests >= 1.0.0
+%if 0%{?suse_version} >= 1500
```

Comment:
AFAIK the same problem exists for RHEL 8.

Reply:
No idea - rhel8 is a) not exactly on my radar as responsible for openSUSE Tumbleweed (future of SLE) and b) not part of any build targets for salt in the OBS project spaces. When rhel 8 becomes available this if would be simple enough to enhance (and possibly change the names if needed - it's not clear if the naming will be exactly the same for the tornado packages)

Reply:
Well, internally we already build it for RES8 and external it is already enabled at https://build.opensuse.org/project/show/systemsmanagement:saltstack:products:testing .
The current problem in OBS is, that a lot of packages cannot be built for CentOS 8 because of the new module concept.

Please just add a `|| 0%{?rhel} >= 8` to the  the places where you check for 
`0%{?suse_version} >= 1500`